build-system/tasks/compile.js

```diff
@@ -227,13 +228,33 @@ function compile(entryModuleFilenames, outputDir,
     // Many files include the polyfills, but we only want to deliver them
     // once. Since all files automatically wait for the main binary to load
     // this works fine.
-    if (options.includePolyfills) {
+    if (options.includeOnlyESMLevelPolyfills) {
+      const polyfillsShadowList = [
+        'array-includes.js',
+        'document-contains.js',
+        'domtokenlist-toggle.js',
+        'math-sign.js',
+        'object-assign.js',
+        'promise.js',
+      ];
       srcs.push(
           '!build/fake-module/src/polyfills.js',
-          '!build/fake-module/src/polyfills/**/*.js'
+          '!build/fake-module/src/polyfills/**/*.js',
+          '!build/fake-polyfills/src/polyfills.js',
+          '!src/polyfills/*.js',
+          'build/fake-polyfills/**/*.js');
+      polyfillsShadowList.forEach(polyfillFile => {
+        fs.writeFileSync('build/fake-polyfills/src/polyfills/' + polyfillFile,
```

Comment:
what was the issue with the errors yesterday?

Reply:
Yesterday we were trying change polyfills.js in `build/fake-modules` whose purpose is to remove all the polyfills including the webcomponents polyfill because every other bundle just uses the polyfill instantiated in `v0.js`.

In this PR I introduced a new folder `build/fake-polyfills` and removed only selected polyfill(everything except web-components one). So now v0-esm will remove all polyfills except `register-element`.
The same cannot be done to `polyfills.js` inside `build/fake-modules` as it will affect the bundles that are being build in parallel.